main.py

```diff
@@ -1,32 +1,25 @@
-from prometheus_client import start_http_server, Summary, Counter, Gauge, Histogram
-import random
 import time
 import requests
 
-# Create a metric to track time spent and requests made.
-REQUEST_TIME = Summary('request_processing_seconds', 'Time spent processing request')
-SIZE_CTR = Counter('size_counter', 'Size counter description')
-SIZE_GG = Gauge('size_gauge', 'Size gauge description')
-HISTOGRAM = Histogram('histie', 'Histogram description')
+from prometheus_client import start_http_server
+
+from model import update_metrics
 
-@REQUEST_TIME.time()
 def pull_metrics():
     """ Pull Memgraph metrics """
     res = requests.get("http://localhost:9092/")
     if res.status_code != 200:
```

Comment:
Handling error codes. Maybe if it is != 200 write the status code to the console it will be easier to debug